### ISSUE TITLE
Method crm.activity.add raises the error

### ISSUE THREAD

**Issue by sbatov** [yarbshk/bitrix24-python3-client#13]
Dear colleague, please help witht the following issue:
I am trying to create the Activity in B24 using webhook. Code:

from pybitrix24 import Bitrix24
bx24 = Bitrix24('consiliummedicum.bitrix24.ru',user_id=7)
bx24.call_webhook('xxxxxxxxxxxxxx', 'crm.activity.add',
   	{'fields':{
   	"TYPE_ID": 5,
   	"SUBJECT": "Новое дело",
   	"RESPONSIBLE_ID": 7,
   	"COMMUNICATIONS": [ { 'VALUE':"+79832322323"}],
   	"OWNER_TYPE_ID": 1,
   	"OWNER_ID": 24657,
   	"COMPLETED": "N",
   	"DESCRIPTION": "Test создания дела"
   	}}
)

The result is the following:
Traceback (most recent call last):
  File "Consilium_8.py", line 14, in <module>
    "DESCRIPTION": "Test создания дела"
  File "C:\ProgramData\Anaconda3\lib\site-packages\pybitrix24\bitrix24.py", line 283, in call_webhook
    data = self._call(url, method, None, params)
  File "C:\ProgramData\Anaconda3\lib\site-packages\pybitrix24\bitrix24.py", line 198, in _call
    data = request(url, query, params)
  File "C:\ProgramData\Anaconda3\lib\site-packages\pybitrix24\requester.py", line 29, in request
    raise PBx24RequestError("Error on request", e)
pybitrix24.exceptions.PBx24RequestError: ('Error on request', <HTTPError 400: 'Bad Request'>)

Tried a variety of changes - does not work. Though other methods (crm.activity.list, crm.activity.get, crm.activity.update) do work properly with the same webhook
Can you spot what's wrong?

**Comment by sbatov** [comment]
I found the root cause. It looks like the COMMUNICATION should be in correlation with TYPE_ID. The following change makes the script operational:
bx24.call_webhook('xxxxxxxxxxxxxx', 'crm.activity.add',
{'fields':{
"TYPE_ID": 2,
"SUBJECT": "Новое дело",
"RESPONSIBLE_ID": 7,
"COMMUNICATIONS": [ { 'VALUE':"+79832322323"}],
"OWNER_TYPE_ID": 1,
"OWNER_ID": 24657,
"COMPLETED": "N",
"DESCRIPTION": "Test создания дела"
}}
)
Am i right?

**Comment by yarbshk** [comment]
hi @sbatov ! nice to hear you again and thanks for interest that you show to the library. 

i've released new version of the library [v1.0.1](https://pypi.org/project/pybitrix24/1.0.1/) that gracefully handles HTTP errors so you can see more descriptive message instead of just "400 Bad request":

```python
>>> data = bx24.call(...)
{'error': '...', 'error_description': '...'}
```

according to your second questions. i think it'll be better to ask in Bitrix24 developers as it's theirs responsibility, and they should be more acknowledged about your issue than me.

**Comment by sbatov** [comment]
First of all - thank you for your prompt comments, it's really helpful as i am struggling with certain issues and your help is priceless. Thank you for version 1.0.1 - i've just installed it, let's see how it works  ;) 
As per this particular issue - you're right, i am probably mistargeting and these are more REST API problems which are the subject of Bitrix Support expertize. Closing the issue therefore.
Take care!
Sergey
